### DISCUSSION TITLE
Install issue

### DISCUSSION THREAD

**Discussion by ChristosGiannopoulos** [nefarius/HidHide#91] [Q&A]
Hello,

I've purchased DualSenseX on steam. I've downloaded the ViGEm driver without issue from within the app. However when trying to install the HidHide driver an error occurs.

![image](https://user-images.githubusercontent.com/116957007/198832402-d1591374-5d3a-413f-afe6-3c6d2cb8ce9b.png)

Is there a solution I'm missing out from the issues page? 

Thanks in advance for any answers.

**Comment by nefarius** [comment]
Do you have a 3rd party anti virus solution running? This can happen if something interferes with it.

**Comment by ChristosGiannopoulos** [comment]
> Do you have a 3rd party anti virus solution running? This can happen if something interferes with it.

No, I only have windows defender on.

**Comment by nefarius** [comment]
Try to delete the folder path mentioned in the dialog box and retry. Other than that, I've got no idea.

**Comment by ChristosGiannopoulos** [comment]
I deleted the folder path but still nothing.
![image](https://user-images.githubusercontent.com/116957007/199077480-f2ff9380-0607-48b1-8af4-cf00756bd637.png)

**Comment by nefarius** [comment]
No clue, mate 😦
